components/apify/apify.app.mjs

```diff
@@ -30,38 +30,24 @@ export default {
       type: "string",
       label: "Actor ID",
       description: "Actor ID or a tilde-separated owner's username and Actor name",
-      async options({ page }) {
-        const { data: { items } } = await this.listActors({
-          params: {
-            offset: LIMIT * page,
-            limit: LIMIT,
-          },
-        });
-
-        return items.map(({
-          id: value, name: label,
-        }) => ({
-          label,
-          value,
-        }));
-      },
-    },
-    userActorId: {
-      type: "string",
-      label: "Actor ID",
-      description: "The ID of the Actor to monitor.",
-      async options({ page }) {
-        const { data: { items } } = await this.listUserActors({
+      async options({
+        page, actorSource,
+      }) {
+        actorSource ??= "recently-used";
+        const listFn = actorSource === "store"
+          ? this.listActors
+          : this.listUserActors;
+        const { data: { items } } = await listFn({
           params: {
             offset: LIMIT * page,
             limit: LIMIT,
           },
         });
 
         return items.map(({
-          id: value, name: label,
+          id: value, title, username, name,
         }) => ({
-          label,
+          label: `${title} (${username}/${name})`,
```

Comment:
Title is optional, so there can be a case that the title is missing in that case, let's show just the tech name `${username}/${name}`